### PR TITLE
Create application

### PR DESCRIPTION
**Pull request by tomve1tch**
-Created application DB to store instances of user_id, post_id and cover letter
-Created application form under the expandable apply option
-Added feature where users who have already applied can no longer apply to that job

### CODE REVIEW

**Comment by saundlj** [comment]
I like the new application fields. be careful with the merge.

File: app/routes.py [hunk 1]
```diff
@@ -67,6 +67,9 @@ def posts():
 @login_required # allows only a logged in user to access account page
 def feed():
     job_posts = Post.query.all()
+    form = FeedApplyForm()
+    if form.validate_on_submit():
+        application = Application(user_id = current_user.id, post_id = )
```
Comment (mjb44552) [inline app/routes.py]: do the parameters in the function "Application" on line 72 all need to have a value assigned to them? will writing just "post_id =" lead to a syntax error?

File: app/routes.py [hunk 2]
```diff
@@ -67,6 +67,9 @@ def posts():
...
     return render_template("FeedPage.html", title = 'Feed',  posts = job_posts )
```
Comment (mjb44552) [inline app/routes.py]: for rendering the feed page what value is the posts variable supposed to be set to? currently we have 2 for loops in the HTML Jinja template. One loops through "posts" (this variable has the dynamic styling applied to it) the other loops through job_posts (which I think was just a variable used to test the page when it was first set up).  I can add this to the issues feed if needed.

File: app/routes.py (hunk 1 above)
Comment (tomve1tch) [reply app/routes.py]: I believe the issue is that you're viewing an outdated commit from when the routes.py file was not fully complete. In the final routes.py file, line 72 is:
application = Application(user_id = current_user.id, cover_letter = form.cover_letter.data, post_id = form.post_id.data)

**Comment by tomve1tch** [comment]
All code seems to be functional. Thank you for the code review Matt :)

**Pull request merged**